main.rb

```diff
@@ -0,0 +1,166 @@
+require_relative 'person'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'rental'
+require_relative 'book'
+
+class App
+  def initialize()
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def list_books
+    if @books.length.positive?
+      @books.each do |book|
+        puts "Title: #{book.title}, Author: #{book.author}"
+      end
+    else
+      puts 'There are no books in the database'
+    end
+
+    puts "\n"
+    main
+  end
+
+  def list_people
+    if @people.length.positive?
+      @people.each do |person|
+        puts "[Person] ID: #{person.id}, Name: #{person.name}, Age: #{person.age}"
+      end
+    else
+      puts 'There are no people in the database'
+    end
+
+    puts "\n"
+    main
+  end
+
+  def create_person
+    print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+    person_type = gets.chomp
+
+    case person_type
+    when '1'
+      create_student
+    when '2'
+      create_teacher
+    else
+      puts "Invalid option \n\n"
+      return main
+    end
+
+    puts "Person created successfully \n\n"
+    main
+  end
+
+  def create_student
+    print 'Age: '
+    age = gets.chomp
+
+    print 'Name: '
+    name = gets.chomp
+
+    print 'Has parent permission? [Y/N]: '
+    parent_permission = gets.chomp
+    parent_permission = parent_permission.downcase == 'y'
+
+    student = Student.new(age, name, permission?: parent_permission)
+    @people.push(student)
+  end
```

Comment:
- [ ] As stated in the project requirement Your console app should behave in the same way as it is in the example.So When I create new student , it throw the below Argument  error , Kindly Fix it.
-  ![Capture](https://user-images.githubusercontent.com/93598189/224128882-340fb0eb-332d-4be2-af0f-77da7f476e79.PNG)

Reply:
Thank you sir, I must have called the method with a bad syntax